core/CMakeLists.txt

```diff
@@ -4,11 +4,16 @@ include_directories(${LIGHTDB_INCLUDE_DIRS})
 
 file(GLOB LIGHTDB_SOURCE_DIRS "*/src")
 foreach(subdir ${LIGHTDB_SOURCE_DIRS})
-  add_subdirectory(${subdir}/..)
+  if (subdir MATCHES "../python/src")
```

Comment:
Can we use a glob that excludes `python/src` instead of doing a `foreach`?  Why are we excluding, anyway?

Reply:
?